src/emailpro/account/order/emailpro-account-order.html

```diff
@@ -48,20 +48,10 @@
                               data-i18n-static="exchange_ACTION_order_accounts_step1_number_not_valid"></span>
                     </div>
                 </div>
-                <div class="oui-checkbox">
-                    <input class="oui-checkbox__input"
-                           type="checkbox" id="legalWarning"
-                           data-ng-model="valid.legalWarning">
-                    <label class="oui-checkbox__label-container" for="legalWarning">
-                        <span class="oui-checkbox__label"
-                              data-ng-bind-html="tr('exchange_ACTION_add_account_legal_warning')"></span>
-                        <span class="oui-checkbox__icon">
-                            <span class="oui-icon oui-icon-checkbox-unchecked" aria-hidden="true"></span>
-                            <span class="oui-icon oui-icon-checkbox-checked" aria-hidden="true"></span>
-                            <span class="oui-icon oui-icon-checkbox-checkmark" aria-hidden="true"></span>
-                        </span>
-                    </label>
-                </div>
+                <oui-checkbox id="legalWarning"
+                    model="valid.legalWarning"
```

Comment:
`data-` for non HTML5 attributes

Reply:
What's the deal with `model` ? Couldn't you use `data-ng-model` ?  It's a candid question, I'm not a lot familiar with components

Reply:
For `data-`, see https://www.w3.org/TR/2016/WD-custom-elements-20161013/#customized-built-in-element. But we could add it for harmonizing the attributes. To be discussed ;)

If we have `ng-model` on and in the component, it doubles the element in the form validation. To avoid this, we rename the attribute `model`. Angular doesn't need to act on the component wrapper :)